### DISCUSSION TITLE
Ace Link

### DISCUSSION THREAD

**Discussion by lef0u** [elrumo/macOS_Big_Sur_icons_replacements#1256] [Icon request] [answered]
Can someone please make a Big Sur style icon for [Ace Link](https://github.com/blaise-io/acelink)?

**Comment by cake03** [comment, marked as answer]
<img src="https://user-images.githubusercontent.com/80169337/125477158-a9fbdd24-6172-45fe-a1de-d43da94769b2.png" height="512" />

**Reply by lef0u** [reply]
Thanks a lot! ❤️
